easybuild/easyconfigs/h/HDF5/HDF5-1.8.13-intel-2015a.eb

```diff
@@ -0,0 +1,26 @@
+name = 'HDF5'
+version = '1.8.13'
```

Comment:
this PR only includes existing easyconfigs with a new toolchain, so not touching versions of dependencies